composer.json

```diff
@@ -14,6 +14,7 @@
         }
     ],
     "require": {
+        "php": "^5.5 || ^7.0",
```

Comment:
Because `,` is a logical `AND` and `||` is the logical `OR` which is needed here :)
https://getcomposer.org/doc/articles/versions.md#range